lab/lab1/part4-data-transformation.html

```diff
@@ -99,7 +111,13 @@
 
     ===================== */
 
+    function create_marker (bike_data) {
+      for (i = 0; i < bike_data.length; i = i + 1) {
+        L.marker([bike_data[i][1].lat, bike_data[i][0]]).addTo(map).bindPopup(bike_data[i][2]).openPopup()
```

Comment:
`.lat` <- is this intended?

Reply:
my bad - that was unintended. I've removed it but the markers still don't seem to show up...

Reply:
Consider peppering your code with `console.log` statements to verify the behavior is as you'd expect. On the surface, things are looking approximately correct so I"m sure you're close

Reply:
Also: consider comparing your work to other Pull Requests to see if there's anything obviously different